async-websocket-wlpcfg/pom.xml

```diff
@@ -86,118 +68,33 @@
                         <default.http.port>${testServerHttpPort}</default.http.port>
                         <default.https.port>${testServerHttpsPort}</default.https.port>
                     </bootstrapProperties>
-                    <install />
+                    <stripVersion>true</stripVersion>
+                    <install/>
```

Comment:
Don't forget you were going to add a `version` parameter to this.  Did that work?